tensorflow/workspace.bzl

```diff
@@ -318,6 +318,9 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
         name = "com_github_googlecloudplatform_google_cloud_cpp",
         sha256 = "839b2d4dcb36a671734dac6b30ea8c298bbeaafcf7a45ee4a7d7aa5986b16569",
         strip_prefix = "google-cloud-cpp-1.14.0",
+        repo_mapping = {
```

Comment:
Let's also split changes to this file into a separate PR.

Reason: we need to do additional changes internally to import this and it is better to have that in a single purpose changelist so it is easy to rollback if needed